package.json

```diff
@@ -24,6 +24,7 @@
     "@ctrl/ts-base32": "^1.2.4",
     "@types/app-root-path": "^1.2.4",
     "@types/cors": "^2.8.8",
+    "@types/mongodb": "^3.6.17",
```

Comment:
Isn't this not necessary now?

Reply:
this is necessary because we have used this.